OpsCenter 6.8.32 Release (#125)
datastax/release-notes: OpsCenter_6.8_Release_Notes.md

# Release notes for OpsCenter

# Release Notes for OpsCenter 6.8.32
11 Jan 2024

## Backup Service
* Add the missing key to details-sstables so opsc won't raise an incorrect error as backup failure (OPSC-17423)
* Improved test cases to add more value to fetch event store. (OPSC-17419)

> * Clean stale backup/snapshot data when previous cleanup or the job failed in Sync Retry. (OPSC-17186)

Reviewer: ```suggestion
* Clean-up stale backup/snapshot data when previous cleanup job failed or the job failed during Sync Retry step. (OPSC-17186)
```

> * Fixed bugs when running a backup using {{allow_one_failure}} when one agent is down. (OPSC-17421)

Reviewer: ```suggestion
* Fixed bugs when running a backup using `{{allow_one_failure}}` when one agent is down. (OPSC-17421)
```

Reviewer: I know it's too late in the game, but shouldn't we name that field as `{{allow_one_agent_failure}}` instead for clarity?

Reviewer: @orion104 thoughts?

## Backup Service,UI
* Added Commitlog backups cache refresh button in commitlogs dialog. (OPSC-17403)

## Core

> * Publish failover and missing heartbeat after heartbeat_reply_period through event services (OPSC-17212)

Reviewer: ```suggestion
* Publish failover and missing heartbeat after `heartbeat_reply_period` through event services (OPSC-17212)
```

## Core,TestEng

> * Scripts for load testing stomp service (OPSC-17443)

Reviewer 2: Remove completely. We only need release notes for code changes that are customer facing. Tests and internal tool don't need release notes.

> * Scripts for load testing stomp service (OPSC-17443)

Reviewer 2: Remove completely. We only need release notes for code changes that are customer facing. Tests and internal tool don't need release notes.

## UI

> * Limit session to User Agent and IP (OPSC-17278)

Reviewer: limit to how much?

Author: single session
